content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -109,12 +109,12 @@ The following methods exist for installing kubectl on macOS:
 1. Test to ensure the version you installed is up-to-date:
 
    ```bash
-   kubectl version --client
+   kubectl version
```

Comment:
This change is not helpful. We want to avoid trying to query a Kubernetes cluster.